pkg/sensors/handler.go

```diff
@@ -181,6 +182,46 @@ func (h *handler) listTracingPolicies(op *tracingPolicyList) error {
 	return nil
 }
 
+func (h *handler) disableTracingPolicy(op *tracingPolicyDisable) error {
+	col, exists := h.collections[op.name]
+	if !exists {
+		return fmt.Errorf("tracing policy %s does not exist", op.name)
+	}
+
+	if !col.enabled {
+		return fmt.Errorf("tracing policy %s is already disabled", op.name)
+	}
+
+	err := col.unload()
+	if err != nil {
+		col.err = fmt.Errorf("failed to unload tracing policy: %w", err)
+		return err
+	}
+
+	col.enabled = false
+	h.collections[op.name] = col
+	return nil
+}
+
+func (h *handler) enableTracingPolicy(op *tracingPolicyEnable) error {
+	col, exists := h.collections[op.name]
+	if !exists {
+		return fmt.Errorf("tracing policy %s does not exist", op.name)
+	}
+
+	if col.enabled {
+		return fmt.Errorf("tracing policy %s is already enabled", op.name)
+	}
+
+	if err := col.load(h.bpfDir, h.mapDir); err != nil {
+		return err
+	}
+
+	col.enabled = true
+	h.collections[op.name] = col
+	return nil
+}
```

Comment:
Will the policyID part will gracefully work with enable/disable?

Reply:
I don't see why not. The policyfilter map will not be affected and the policy id will not change when we unload things.